### DISCUSSION TITLE
App/Program Redundancy

### DISCUSSION THREAD

**Discussion by lbeck02** [Open-Shell/Open-Shell-Menu#2034] [Q&A]
I've been using OS for several months and like it.  But I'm finding considerable redundancy between "Apps" and "Programs".  More than the numerous versions of Windows (3.1 - 11).

Is there a way to combine the 2 lists into one?

**Comment by veermaharaj** [comment]
Ditto. How can both of these things be combined?

Started using UniGetUI and stuff that install from there are ending up in the APPs folder.

Thanks and regards.
